### DISCUSSION TITLE
Github, request rejected

### DISCUSSION THREAD

**Discussion by Mar30co** [community/community#68519] [GitHub Education] [answered]
### Hi everyone,

ello,

I am a student at CPT Locarno in Switzerland, and our school is registered with the GitHub School Program. I have made multiple requests, submitting my information along with a photo of my school ID and school certificate, but I have consistently received rejections without any explanations. Can someone please assist me?

Tanks.
Mar30co

**Comment by vishalkrishna8** [comment, marked as answer]
Hello @Mar30co 
If you're facing rejections from the GitHub School Program and need assistance:

1.Double-check eligibility.
2.Contact GitHub Support for explanations and help.
3.Ensure all information is accurate.
4.Check spam folder for emails from GitHub.
5.Involve your school if necessary.

Let me know if this was helpful to you!
